clients/callback_client.go

```diff
@@ -181,6 +198,15 @@ func (ts TranscodeStatus) String() string {
 
 // The various status messages we can send
 
+type RecordingEvent struct {
+	When        string `json:"when"`
+	StreamId    string `json:"streamId"`
+	RecordingId string `json:"recordingId"`
+	Hostname    string `json:"hostName"`
```

Comment:
For consistency with the other Catalyst APIs, I believe this should use `snake_case`
```suggestion
	StreamId    string `json:"stream_id"`
	RecordingId string `json:"recording_id"`
	Hostname    string `json:"host_name"`
```